### DISCUSSION TITLE
Color of tabbar background

### DISCUSSION THREAD

**Discussion by ghost** [kovidgoyal/kitty#3987] [Q&A]
I can color the active and inactive tabs but is it possible to set the color of the unused part of the tabbar to the right of the tabs? I want to give it another color than black so that the bar looks to go the whole way to the right.

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.tab_bar_background

**Reply by marlun** [reply]
Ah, sorry for missing that, worked great!
